gcc/config/aarch64/aarch64.cc

```diff
@@ -7080,6 +7217,13 @@ aarch64_layout_arg (cumulative_args_t pcum_v, const function_arg_info &arg)
   if (pcum->aapcs_arg_processed)
     return;
 
+#if defined (TARGET_AARCH64_MS_ABI)
```

Comment:
The same, not sure TARGET_AARCH64_MS_ABI check is needed.

Reply:
Not really, it's rather just a performance optimization. I am not sure what the maintainers prefer as I've seen such optimizations throughout the codebase?

Reply:
Removing it whould mean to remove the same condition for `aarch64_ms_variadic_abi_layout_arg`. Do you think that all the target specific functions hold be defined for all targets, effectively removing most of the `TARGET_AARCH64_MS_ABI` check?

Reply:
Yes, it should be fine to define them without guards if it is not breaking other targets.

Reply:
The check for `TARGET_AARCH64_MS_ABI` removed in ce4228b.